api-docs/rst/dev-guide/api-operations/methods/put-update-policy-v1.0-tenantid-groups-groupid-policies-policyid.rst

```diff
@@ -189,20 +189,20 @@ This table shows the body parameters for the request:
 |                            |             |exclusive with the ``cron`` parameter.     |
 +----------------------------+-------------+-------------------------------------------+
 |scalingPolicies.[*].\       |Number       |The percent change to make in the number   |
-|**changePercent**           |*(Optional)* |of servers in the scaling group. If this   |
-|                            |             |number is positive, the number of servers  |
+|**changePercent**           |*(Optional)* |of units in the scaling group. If this     |
+|                            |             |number is positive, the number of units    |
 |                            |             |increases by the given percentage. If this |
 |                            |             |parameter is set to a negative number, the |
-|                            |             |number of servers decreases by the given   |
+|                            |             |number of units decreases by the given     |
 |                            |             |percentage. The absolute change in the     |
```

Comment:
A positive value increase the number of units by the specified percentage. A negative value decreases the number of units.